tests/integration/test_orders_cli.py

```diff
@@ -540,20 +540,13 @@ def test_cli_orders_create_basic_success(expected_ids,
     request_file = write_to_tmp_json_file(json.loads(request_result.output),
                                           'orders.json')
 
-    # Invoke the create call
-    invoke(['create', str(request_file)])
+    result = CliRunner().invoke(cli.main,
+                                ['orders', 'create', str(request_file)],
+                                catch_exceptions=False)
```

Comment:
@kevinlacaille `catch_exceptions=False` is good for debugging, but we should remove this option (default to True). We kinda depend on it to get the right exit code in tests.